src/test/java/com/marklogic/kafka/connect/source/ReadRowsViaOpticSerializedTest.java

```diff
@@ -0,0 +1,71 @@
+package com.marklogic.kafka.connect.source;
+
+import com.marklogic.client.document.XMLDocumentManager;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.FileHandle;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ReadRowsViaOpticSerializedTest extends AbstractIntegrationSourceTest {
+
+    @Test
+    void testRowBatcherTask() throws InterruptedException {
+        loadMarkLogicTestData();
+
+        RowBatcherSourceTask task = startSourceTask(
+            MarkLogicSourceConfig.DMSDK_BATCH_SIZE, "1",
+            MarkLogicSourceConfig.SERIALIZED_QUERY, AUTHORS_OPTIC_SERIALIZED,
+            MarkLogicSourceConfig.TOPIC, AUTHORS_TOPIC
+        );
+
+        List<SourceRecord> newSourceRecords = task.poll();
+
+        Assertions.assertEquals(15, newSourceRecords.size());
+        assertTopicAndSingleValue(newSourceRecords, AUTHORS_TOPIC);
+    }
+
+    @Test
+    void noRowsReturned() throws InterruptedException {
```

Comment:
I think this test is simple enough to keep it as-is, no need to bubble anything up to the parent class.